snow/state.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	Initializing = iota
+	Initializing State = iota
```

Comment:
This is needed to enable using `require.Equal` rather than `require.EqualValues` (or explicitly casting)